docs/local_resource.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Run Local and/or Occasional Workflows with Local Resource
+layout: docs
+---
+Each entry in your Tilt sidebar is a **resource**---a unit of work managed by Tilt. (Until
```

Comment:
"Until this point" / "have been" make sense in a blog with a publish date, but less sense in a doc

Reply:
fair nough